.github/workflows/create_tag.yml

```
@@ -1,31 +1,32 @@
-      - name: Checkout
-        uses: actions/checkout@v2
+      - uses: actions/checkout@v4
         with:
           ref: release
           fetch-depth: 0
-      - name: Get last commit message
-        id: commit_message
+
+      - name: Derive tag from commit message
+        id: tag
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2001:style:2:7: See if you can use ${variable//search/replace} instead

(shellcheck)